aci/compose.go

```diff
@@ -100,7 +101,7 @@ func (cs aciComposeService) warnKeepVolumeOnDown(ctx context.Context, projectNam
 		if v.AzureFile == nil || v.AzureFile.StorageAccountName == nil || v.AzureFile.ShareName == nil {
 			continue
 		}
-		fmt.Printf("WARNING: fileshare \"%s/%s\" will NOT be deleted. Use 'docker volume rm' if you want to delete this volume\n",
+		fmt.Fprintf(os.Stderr, "WARNING: fileshare \"%s/%s\" will NOT be deleted. Use 'docker volume rm' if you want to delete this volume\n",
```

Comment:
Do we still want to print out the stderr contents if we're in quiet mode? What does docker/cli do?

cc @thaJeztah

Reply:
good point

Reply:
Hm... will have to check. Not sure what would be most expected. 🤔. I guess it's a "warning", so not a _critical_ message, but not sure how important it is to let the user know